### ISSUE TITLE
Wrong field order

### ISSUE THREAD

**Issue by jiangweiatgithub** [alingse/jsoncsv#46]
When I use the command line conver the following lines:
{"english": "Slowly and not without struggle, America began to listen.", "chinese": "美国缓慢地开始倾听，但并非没有艰难曲折。"}
{"english": "I didn't own a Thesaurus until four years ago and I use a small Webster's dictionary that I'd bought at K-Mart for 89 cents.", "chinese": "直到四年前我才有了一本词典。我使用的是用89美分在K市场里买来的一本韦氏小词典。我从来不使用单词处理程序。"}
{"english": "portlet, you must write three short deployment descriptors: web.xml, portlet.xml, and geronimo-web.xml. (Some of these may have been generated by your IDE.)", "chinese": "portlet 之后，您必须编写三个简短的部署描述符：web.xml、portlet.xml 和 geronimo-web.xml（这其中的一些文件可能已经由 IDE 生成）。"}

 I get the result in an inversed order:
chinese,english
美国缓慢地开始倾听，但并非没有艰难曲折。,"Slowly and not without struggle, America began to listen."
直到四年前我才有了一本词典。我使用的是用89美分在K市场里买来的一本韦氏小词典。我从来不使用单词处理程序。,I didn't own a Thesaurus until four years ago and I use a small Webster's dictionary that I'd bought at K-Mart for 89 cents.
portlet 之后，您必须编写三个简短的部署描述符：web.xml、portlet.xml 和 geronimo-web.xml（这其中的一些文件可能已经由 IDE 生成）。,"portlet, you must write three short deployment descriptors: web.xml, portlet.xml, and geronimo-web.xml. (Some of these may have been generated by your IDE.)"

**Comment by alingse** [comment]
yes, see this issue https://github.com/alingse/jsoncsv/issues/45 here.

对，最近我看看怎么改一下好。
